world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
```

Comment:
You can nest dicts. But my preference is to remove `skill_manager` and then set it up like 

in world/abilities.py:

{name: sneak, .....}

{name: leadership, .....}

We can iterate this file at runtime to construct singleton `Abilities`.